native/components/message-markdown.react.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import { Component } from 'react';
+import * as React from 'react';
+import { View, ViewPropTypes } from 'react-native';
+import { merge, pick } from 'lodash';
```

Comment:
1. Let's use the `lodash/fp` family instead of the older `lodash` one
2. We should never import directly from `lodash` or `lodash/fp` because it bloats bundle size. We can just import the functions we need one-by-one

Check out how I import from `lodash/fp` elsewhere in the project

Reply:
Will remember for the future, since we don't need `lodash` now with the rules coming from one place